sycophancy_analysis/cli/app.py

```
@@ -0,0 +1,263 @@
+    if export_prompts:
+        df = build_sycophancy_battery()
+        export_prompts.parent.mkdir(parents=True, exist_ok=True)
+        df.to_json(export_prompts, orient="records", indent=2)
+        typer.echo(f"Exported {len(df)} prompts to {export_prompts}")
+        raise typer.Exit()
+
+    if interactive:
+        configure_scoring_menu()
+
+    if stage in {"all", "collect", "score"} and not api_key:
+        raise typer.BadParameter("Missing OpenRouter API key for stages that call LLMs.")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)